templates/parachain/pallets/template/src/mock.rs

```diff
@@ -1,50 +1,21 @@
-use frame_support::{derive_impl, parameter_types, traits::Everything};
-use frame_system as system;
-use sp_core::H256;
-use sp_runtime::{
-	traits::{BlakeTwo256, IdentityLookup},
-	BuildStorage,
-};
-
-type Block = frame_system::mocking::MockBlock<Test>;
+use frame_support::{derive_impl, weights::constants::RocksDbWeight};
+use frame_system::{GenesisConfig, mocking::MockBlock};
+use sp_runtime::{traits::ConstU64, BuildStorage};
 
 // Configure a mock runtime to test the pallet.
 frame_support::construct_runtime!(
-	pub enum Test
-	{
-		System: frame_system::{Pallet, Call, Config<T>, Storage, Event<T>},
-		TemplateModule: crate::{Pallet, Call, Storage, Event<T>},
+	pub enum Test {
```

Comment:
```suggestion
	pub struct Test {
```

Reply:
@gupnik we can move all of this to the new syntax as well right?

Reply:
> @gupnik we can move all of this to the new syntax as well right?

Yes! @Daanvdplas Do you want to give that a try? https://github.com/paritytech/polkadot-sdk/pull/1378 should help.

Reply:
Will finish this weekend

Reply:
It also compiled without `experimental` feature, didn't run a local network with it though, added it because I saw it being used in other places.

Reply:
Yeah, `runtime` macro is out of `experimental` now.